__tests__/app.test.js

```diff
@@ -0,0 +1,77 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data')
+const db = require('../db/connection.js')
+
+beforeEach(() => seed(data))
+
+afterAll(() => db.end());
+
+describe("GET /*", () => {
+  test("Status 404 when given invalid path", () => {
+      return request(app)
+        .get("/badpath")
+        .expect(404)
+        .then((response) => {
+          expect(response.error.text).toBe("Path not found!")
+        })
+  })
+})
+
+describe("GET /api/topics", () => {
+    test("Status 200", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+    });
+    test("returns an array of objects with properties 'slug', 'description", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+        .then((response) => {
+            expect(response.body).toHaveLength(3);
+            response.body.forEach((cell) => {
```

Comment:
I'd maybe rename 'cell' to be 'topic' just to be more clear about what exactly is within this array 😄